inference-engine/cmake/dependencies.cmake

```diff
@@ -125,6 +125,14 @@ if (THREADING STREQUAL "OMP")
     update_deps_cache(OMP "${OMP}" "Path to OMP root folder")
     log_rpath_from_dir(OMP "${OMP}/lib")
     debug_message(STATUS "intel_omp=" ${OMP})
+    
+    if(WIN32)
+        file(GLOB_RECURSE source_list "${OMP}/*.dll")
+    elseif(LINUX OR APPLE)
```

Comment:
`LINUX OR APPLE` -> `UNIX`

Reply:
The proper way is to use https://cmake.org/cmake/help/v3.0/variable/CMAKE_SHARED_LIBRARY_SUFFIX.html to have an unified code here.

Reply:
BTW, we don't ship packages with OMP.